.github/workflows/test.yml

```diff
@@ -13,14 +13,18 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build the project
         run: |
-          docker-compose -f docker-compose.yml up -d --build
-          docker-compose -f docker-compose.yml exec -T backend pip install -r requirements-dev.txt
+          docker-compose up -d --build backend
+          docker-compose exec -T backend pip install -r requirements-dev.txt
       - name: Lint the code
         run: |
           docker-compose exec -T backend black --check .
           docker-compose exec -T backend flake8
           docker-compose exec -T backend ./manage.py makemigrations --check --dry-run --no-input
       - name: Run pytest
         run: docker-compose exec -T backend pytest --no-cov-on-fail --cov --create-db -vv
+      - name: Copy .coverage
+        run: docker cp $(docker ps -q backend):/app/.coverage .coverage
```

Comment:
This should be `docker-compose ps -q backend` I think.

Ugly workaround, but I don't see a better way either

Reply:
Thanks!